deployment/fedora-package-x64/package.sh

```diff
@@ -18,10 +18,15 @@ output_dir="`pwd`/pkg-dist"
 pkg_src_dir="`pwd`/pkg-src"
 current_user="`whoami`"
 image_name="jellyfin-rpmbuild"
+docker_sudo=""
```

Comment:
What's the usecase for doing `sudo` here?

Reply:
It's to just run the `docker` commands with `sudo` instead of running the whole script as root since RH distros don't have a `docker` group on the system by default since it is equal to [root privileges](https://www.projectatomic.io/blog/2015/08/why-we-dont-let-non-root-users-run-docker-in-centos-fedora-or-rhel/).